docker/Dockerfile-version

```diff
@@ -1 +1 @@
-1_14
+1_15
```

Comment:
Could you rebase this on master first? Otherwise you're not including the change made in https://github.com/mobilecoinfoundation/mobilecoin/pull/590.

Reply:
:+1: thank you, didn't see that